application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -130,10 +130,19 @@
     var submitInput = saveForm.find("input[type='submit']");
     saveForm.submit(function(event) {
       event.preventDefault();
+      // Transform the form data into a JSON so the Vote object can be deserialized properly server side.
```

Comment:
It's very clear how you create the Vote object. I would have expected something like:

    var vote = {proposals: [...]};